MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -15,18 +15,17 @@ internal static class Sdl
     public static int Minor;
     public static int Patch;
 
-    private static unsafe string GetString(IntPtr handle)
+    private static string GetString(IntPtr handle)
     {
         if (handle == IntPtr.Zero)
             return "";
 
-        var ptr = (byte*) handle;
-        while (*ptr != 0)
-            ptr++;
+        int len = 0;
+        while (Marshal.ReadByte(handle, len) != 0)
```

Comment:
Has this been tested on Mac, Linux and Windows? I have a vague recollection that I tried this technique (or something similar) and it didn't work on one of the platforms. I could be wrong though.